src/app/ApplicationTemplate.Shared.Views/Content/Diagnostics/DiagnosticsOverlay.xaml

```diff
@@ -355,14 +354,46 @@
 					
 					<!--Label & Value -->
 					<Run Text="UC:" />
-					<Run Text="{Binding UncollectedCommands}" />
+					<Run Text="{Binding Counters.UncollectedCommands}" />
 					
 					<!-- Tool Tip -->
 					<ToolTipService.ToolTip>
 						<ToolTip Style="{StaticResource DiagnosticsCounterToolTipStyle}"
 								 Content="Number of uncollected commands (Disposed - Destroyed)" />
 					</ToolTipService.ToolTip>
 				</TextBlock>
+
+				<!-- Private Memory Size -->
```

Comment:
I think you should align both memory items to the right to that the "MB" are aligned.